### ISSUE TITLE
Issue installing with JupyterLab

### ISSUE THREAD

**Issue by andrew-barnhart** [NotNANtoN/diffusion_gen#7]
I am attempting to install this in a JupyterLab notebook on JarvisLabs.ai. I put all the files into the home directory and run
`ipython3 diffuse.py`

I get this error first:
<img width="629" alt="Screen Shot 2022-02-09 at 2 09 42 PM" src="https://user-images.githubusercontent.com/94346601/153303268-5b1657d9-efa7-4b48-9f50-fe92e508bd65.png">

When I run it again, I get the following:
<img width="798" alt="Screen Shot 2022-02-09 at 2 10 42 PM" src="https://user-images.githubusercontent.com/94346601/153303531-edbe34ad-3ebb-4d51-baf1-127b18a6d433.png">
<img width="802" alt="Screen Shot 2022-02-09 at 2 48 52 PM" src="https://user-images.githubusercontent.com/94346601/153303634-11171c03-5362-4756-b29b-5a1ded3b2aa2.png">
<img width="749" alt="Screen Shot 2022-02-09 at 2 49 06 PM" src="https://user-images.githubusercontent.com/94346601/153303645-25e56f86-65a1-4efd-8e57-82a3ae6d0d88.png">

Any suggestions? I am pretty new to this but want to be able to use an A100 consistently, which you can't do in Colab.

My directory looks like this. I noticed that /out_diffusion/models is empty
<img width="371" alt="Screen Shot 2022-02-09 at 2 47 53 PM" src="https://user-images.githubusercontent.com/94346601/153303723-81baa449-5675-4b3b-899d-09f06a616cbd.png">

**Comment by andrew-barnhart** [comment]
Update: I added the 512x512_diffusion_uncond_finetune_008100.pt and secondary_model_imagenet_2.pth models from the Disco Diffusion 4.1 package and it seems to be working now! Not sure why the clone wouldn't work.

**Comment by NotNANtoN** [comment]
Hi, glad you got it working. I'm also not sure why this did not work for you - did you download the .pth files from the links manually now and put them in the repository?

**Comment by NotNANtoN** [comment]
Closing for now...

**Comment by andrew-barnhart** [comment]
Hi, sorry for the late response. I searched the web for the models & I found the primary one [here](https://huggingface.co/lowlevelware/512x512_diffusion_unconditional_ImageNet/blob/main/512x512_diffusion_uncond_finetune_008100.pt) and I think I pulled the secondary one from my Google Drive because I had run the Colab notebook for Disco Diffusion 4.1 and it was saved there. I could not get SuperRes to work in the same way, but I can run that separately.
